cabal.project

```diff
@@ -3,4 +3,9 @@ source-repository-package
   location: https://github.com/colinhect/hsnoise
   tag: 4ccff11dea7e8d94e6a5fcaf8f43857bd65bd72d
 
+source-repository-package
+  type: git
+  location: https://github.com/jtdaugherty/brick.git
```

Comment:
Just based on comparing with the existing `source-repository-package`, maybe the `.git` should not be there?

Reply:
Did not work, could it be that the `cabal.project` file is not even there?

Otherwise, I do not see why the build did not download the commit. :confused:

Reply:
Hmm, I don't know.  I guess if CI was building against the version of `hsnoise` on Hackage instead of getting it from github, we would never have noticed before, since it just affects the way randomly-generated worlds look.

Reply:
Well, `rm -f cabal.project cabal.project.local` in **generate cabal.project** is kind of the obvious villain. :facepalm: